main.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"encoding/csv"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+)
+
+type Quote = struct {
+	ID         string
+	NationalID string
+	Name       string
+	Title      string
+	Message    string
+	Pokedex    string
+}
+
+func home(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	view := `
+		<h1>Welcome to the PokéLimbo!</h1>
+		<p>
+			Feel free to go to the
+			<a title="/pokehell" href="/pokehell">PokéHell</a>
+			... if you dare.
+		</p>
+	`
+	log.Print("Hit: Home")
+	fmt.Fprintf(w, view)
+}
+
+func pokehell(w http.ResponseWriter, r *http.Request) {
+	log.Print("Hit: PokéHell")
+	id := r.URL.Query().Get("id")
+	db, err := os.Open("db.csv")
+	if err != nil {
+		log.Fatal(err)
+	}
+	reader := csv.NewReader(db)
+	reader.Comment = '#'
+	lines, err := reader.ReadAll()
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	var quote Quote
+
+	var quotesList []Quote
+
+	for _, currentQuote := range lines {
+		quote.ID = currentQuote[0]
+		quote.NationalID = currentQuote[1]
+		quote.Name = currentQuote[2]
+		quote.Title = currentQuote[3]
+		quote.Message = currentQuote[4]
+		quote.Pokedex = currentQuote[5]
```

Comment:
```suggestion
	var quotesList []Quote

	for _, currentQuote := range lines {
	  quote := Quote{
	    ID: currentQuote[0],
	  	NationalID: currentQuote[1],
	  	Name: currentQuote[2],
	  	Title: currentQuote[3],
  		Message: currentQuote[4],
	  	Pokedex: currentQuote[5],
	  }
```

We can drop the declaration of the previous `quote` and instead declare it inside the loop.